gitlab/Makefile

```diff
@@ -5,5 +5,5 @@ include ../_scripts/Makefile.projects
 config:
 	@${BIN}/reconfigure_ask ${ENV_FILE} GITLAB_TRAEFIK_HOST "Enter your gitlab domain name" "gl.${ROOT_DOMAIN}"
 	@${BIN}/reconfigure_ask ${ENV_FILE} APP_NAME "Enter the service description" "git thing"
-	@${BIN}/reconfigure ${ENV_FILE} POSTGRES_PASS=$(shell openssl rand -hex 45)
-	@${BIN}/reconfigure ${ENV_FILE} GITLAB_ROOT_PASSWORD=$(shell openssl rand -hex 45)
+	@[[ -z "$$(dotenv -f ${ENV_FILE} get POSTGRES_PASS)" ]] && ${BIN}/reconfigure ${ENV_FILE} POSTGRES_PASS=$(shell openssl rand -hex 45) || true
```

Comment:
dotenv should be ${BIN}/dotenv

Reply:
You can now use the [reconfigure_password](https://github.com/EnigmaCurry/d.rymcg.tech/blob/master/_scripts/reconfigure_password) script instead of this.

```
@reconfigure_password ${ENV_FILE} POSTGRES_PASS
```